assets/css/stripeStyle.css

```diff
@@ -544,7 +549,8 @@ table tr {
 }
 
 .error-message {
-	color: #080808;
+	color: #D32F2F;
```

Comment:
Codacy found an issue: [Expected "#D32F2F" to be "#d32f2f" (color-hex-case)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5972218)